packages/camera/camera_platform_interface/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   flutter:
     sdk: flutter
   meta: ^1.3.0
-  plugin_platform_interface: ^2.0.0
+  plugin_platform_interface: '>=2.1.0 <4.0.0'
```

Comment:
Since the final version of the platform interface chance doesn't require any breaking changes (the deprecation after this all lands is just a minor bump per semver and standard Dart package practice), these should all be `^2.1.0` instead.

Reply:
That is what I had in a previous version of this PR, and I've reverted back to it.